data/mods/Aftershock/player/professions.json

```diff
@@ -85,30 +85,6 @@
       "female": [ "sports_bra", "boy_shorts" ]
     }
   },
-  {
-    "type": "profession",
-    "id": "afs_speed_freak",
-    "name": "Speed Freak",
-    "description": "Were you a mercenary, a criminal, a pirate or a very dedicated adrenaline junkie?  Whatever you were before, now you've got a speed freak rig pumping drugs into your veins 24/7.  Statistically you'll die young, your body burned out by all the drugs, but what other chance did you have to become an immortal legend?  None.",
-    "points": 3,
-    "CBMs": [ "afs_bio_chemical_enhancement_rig", "bio_adrenaline", "bio_metabolics", "bio_power_storage_mkII" ],
-    "skills": [
-      { "level": 1, "name": "gun" },
-      { "level": 1, "name": "cutting" },
-      { "level": 1, "name": "melee" },
-      { "level": 1, "name": "dodge" }
-    ],
-    "traits": [ "STR_UP_3", "UPLIFTED", "THRESH_URSINE", "URSINE_FUR", "CANNIBAL", "HUGE_OK" ],
```

Comment:
Ooof I can't believe I left in traits from the thing I'd copied from to create the base format.  Glad you are doing this.

Reply:
Its happened to me more than once haha.